simdjson.stub.php

```
@@ -224,6 +224,11 @@
  * @cvalue SIMDJSON_PHP_ERR_INVALID_PHP_PROPERTY
  */
 const SIMDJSON_ERR_INVALID_PROPERTY = UNKNOWN;
+/**
+ * @var int
+ * @cvalue SIMDJSON_PHP_ERR_INPUT_SIZE_EXCEEDS
+ */
+const SIMDJSON_PHP_ERR_INPUT_SIZE_EXCEEDS = UNKNOWN;
```

PHPStan (2.1.46)

[error] Constant UNKNOWN not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)